pkgs/development/python-modules/qiskit-aqua/default.nix

```diff
@@ -97,7 +97,10 @@ buildPythonPackage rec {
       >> qiskit/optimization/__init__.py
   '';
 
-  postInstall = "rm -rf $out/${python.sitePackages}/docs";  # Remove docs dir b/c it can cause conflicts.
+  # This directory can cause conflicts w/ other packages
+  postInstall = ''
+    rm -rf $out/${python.sitePackages}/docs
```

Comment:
```suggestion
    rm -r $out/${python.sitePackages}/docs
```

Reply:
Addressed in latest force-push. Addressed all instances of ``rm -rf`` in all addressed packages.